openshift/WFLY-15421_OIDC_for_WildFly_s2i_v2.adoc

```diff
@@ -0,0 +1,297 @@
+= OIDC support for WildFly S2I V2 image
+:author:           Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+For WildFly s2i V2 we are removing the keycloack OIDC and SAML client adapters present in the s2i image.
+The only supported integration with keycloak would be based on the `elytron-oidc-client` subsystem and 
+`elytron-oidc-client` Galleon layer.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-15421[WFLY-15421]
+
+=== Related Issues
+
+* OIDC Support for WildFly: https://issues.redhat.com/browse/WFCORE-5178[WFCORE-5178]
+
+* WildFly s2i V2 images: https://issues.redhat.com/browse/WFLY-14936[WFLY-14936]
+
+* WildFly Maven plugin evolutions for the cloud: https://issues.redhat.com/browse/WFLY-14934[WFLY-14934]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [ ] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+* https://github.com/wildfly/wildfly-s2i[WildFly s2i]
+
+=== Relevant Installation Types
+// Remove the x next to the relevant field if the feature in question is not relevant
+// to that kind of WildFly installation
+* [ ] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [ ] Managed domain
+
+* [x] OpenShift s2i
+
+* [ ] Bootable jar
```

Comment:
Is this also impacted?
Could a Bootable Jar image uses the `elytron-oidc-client` layer?

Reply:
This proposal is not bound to bootable JAR, it is wildfly-s2i specific. The introduction of the galleon layer is in https://github.com/wildfly/wildfly-proposals/pull/341